test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -189,14 +189,14 @@ def train_mnist_model(serving_folder_path, ec2_connection, python_invoker):
 def host_setup_for_tensorflow_inference(
     serving_folder_path, framework_version, ec2_connection, is_neuron, is_graviton, model_name, python_invoker
 ):
-    # Installing protobuf at 3.20.* to fix errors in using latest protobuf
-    ec2_connection.run((f"{python_invoker} -m pip install --user -qq -U 'protobuf>=3.20,<3.21'"), hide=True)
+    # Install PIP so we can test
+    ec2_connection.run((f"sudo apt-get update && sudo apt-get install -y python3-pip"), hide=True)
 
-    # Tensorflow 1.x doesn't have package with version 1.15.2 so use only 1.15
+    # Per graviton team, need to install tensorflow from pypi with no version pinning. attempting a pin will result is
+    # pip not finding the version. the internal repo only has a custom Tensorflow 2.6 which is not compatible with TF 2.9+
+    # and this was the recommended action by the Graviton team.
     if is_graviton:
-        # TF training binary is used that is compatible for graviton instance type
-        TF_URL = "https://aws-dlc-graviton-training-binaries.s3.us-west-2.amazonaws.com/tensorflow/2.6.0/tensorflow-2.6.0-cp38-cp38-linux_aarch64.whl"
-        ec2_connection.run((f"{python_invoker} -m pip install --no-cache-dir -U {TF_URL}"), hide=True)
+        ec2_connection.run((f"{python_invoker} -m pip install --no-cache-dir -U tensorflow-cpu-aws"), hide=True)
```

Comment:
Is tensorflow-cpu-aws a valid URI?

Reply:
it's not a uri.. it's an official pypi package that is required for the new TF2.9 testing. Worked this with graviton team on this one and that is the needed package for the test.